knimin/lib/util.py

```diff
@@ -198,6 +198,26 @@ def categorize_age(x):  # noqa
     return age_cat
 
 
+def correct_age(age, height, weight, etoh):
+    """Infers incorrect ages and incorrectly classified babies"""
+    # Checks the logic for age
+    if age >= 4 and age < 123:
```

Comment:
it's not just babies though, it's also highlanders (people over 123 years old) and time travelers (people with negative ages)